### PR TITLE
A few spelling / grammar fixes

### PR DESCRIPTION
**Pull request by SuborbitalPigeon**
I've found a few spelling and grammar errors, I've fixed these in one commit. 

I also found a few uses of Commonwealth english, things like 'behavio**u**r'. I think it would make sense in-universe to keep these as there will be people from all sorts of brackgrounds, and some would use British forms of spelling and so on. I've made this a separate commit which can be ignored if these changes aren't desired.

### CODE REVIEW

File: data/lang/module-newseventcommodity/en.json [hunk 1]
```diff
@@ -53,15 +53,15 @@
   },
   "FLAVOUR_13_NEWSBODY": {
     "description": "Imply that nobody wants to buy ROBOTS anymore.",
-    "message": "It is now man against machine in the {system} system ({sectorx}, {sectory}, {sectorz}), as \"behavioural malfunction\" causes home consumers to lose faith in robot servants."
+    "message": "It is now man against machine in the {system} system ({sectorx}, {sectory}, {sectorz}), as \"behavioral malfunction\" causes home consumers to lose faith in robot servants."
```
Comment (impaktor) [inline data/lang/module-newseventcommodity/en.json:56]: Not spelling error, we're a clone of a UK game, and most of the native speakers have been from UK.

File: data/lang/module-advice/en.json [hunk 2]
```diff
@@ -89,7 +89,7 @@
   },
   "RUMOUR": {
     "description": "Headline starts with this pre-headline",
-    "message": "RUMOUR"
+    "message": "RUMOR"
```
Comment (impaktor) [inline data/lang/module-advice/en.json:92]: Not a spelling error, UK.

File: data/lang/module-newseventcommodity/en.json [hunk 3]
```diff
@@ -117,7 +117,7 @@
   },
   "FLAVOUR_20_NEWSBODY": {
     "description": "Imply vast abundance of FERTILIZER. Soylent is a reference to the book/movie Soylent Green.",
-    "message": "As {system} ({sectorx}, {sectory}, {sectorz}) enters a phase favourable to the yellow algae found in the atmosphere of a local gas giant, the harvest has be exceptionally good for the fertilizer company Soylent Yellow Inc."
+    "message": "As {system} ({sectorx}, {sectory}, {sectorz}) enters a phase favorable to the yellow algae found in the atmosphere of a local gas giant, the harvest has be exceptionally good for the fertilizer company Soylent Yellow Inc."
```
Comment (impaktor) [inline data/lang/module-newseventcommodity/en.json:120]: not a spelling error.

File: data/lang/ui-core/en.json [hunk 4]
```diff
@@ -2209,7 +2209,7 @@
...
+    "message": "After a long bureaucratic process, I finally got my ship back from the impound, but they had stripped her down to the bare metal, to pay 'the debt' they claimed I had. Not only did they take my favorite coffee brewer, but those bastards also ripped out the hyper drive, so I'm stuck for now in this system, unless I sell whatever is left in the ship, to finance a new drive. But as they say: when life gives you lemons - make lemonade; so even if I'm back at square one, I will think of this as an opportunity instead of as a setback, a second chance. Without coffee. By letting go of what I am, I will become what I might be. The world will turn aside, and let me pass, as it does for those who know where they are going, and I am heading to the stars."
```
Comment (impaktor) [inline data/lang/ui-core/en.json:2212]: not a spelling error

**Review by impaktor — COMMENTED** [review]
Ideally we want to do as few updates to translated strings as possible, to not trigger re-translation in all languages. 

My belief is we're generally following British English. (Also, there's different spellings of UK English in regards to -ize vs. -ise,).

With that said, I think we keep our colour, favourite, behavioural, etc.

**Comment by RacerBG** [comment]
> Ideally we want to do as few updates to translated strings as possible, to not trigger re-translation in all languages.

I'm fine with this. These grammar errors are not present in the translations so a simple save & review does the job.

File: data/lang/module-newseventcommodity/en.json (hunk 1 above)
Comment (SuborbitalPigeon) [reply data/lang/module-newseventcommodity/en.json:56]: That's fine, I'll drop that commit

File: data/lang/module-advice/en.json (hunk 2 above)
Comment (SuborbitalPigeon) [reply data/lang/module-advice/en.json:92]: Same

File: data/lang/module-newseventcommodity/en.json (hunk 3 above)
Comment (SuborbitalPigeon) [reply data/lang/module-newseventcommodity/en.json:120]: Again

File: data/lang/ui-core/en.json (hunk 4 above)
Comment (SuborbitalPigeon) [reply data/lang/ui-core/en.json:2212]: Yup

**Comment by Web-eWorks** [comment]
> My belief is we're generally following British English.

I don't think there's anything necessarily set in stone regarding use of one or the other, but the existing precedent in our language strings is for the British spelling rather than the American spelling.

That being said, thanks for the spelling fixes! It's definitely worth a bit of translation workload to address mis-spelled words, especially when we have (at least) several weeks for the translation churn to be looked at before a release.

**Comment by fluffyfreak** [comment]
It's because it was started by TomM (UK), I'm UK, RobN (AUS), plus a few others... also given my other career it's been nice to work on a game where we could keep the UK spellings 😆

**Comment by Keeper1st** [comment]
Heck, I have a whole crapload of such fixes if you want 'em, as well rewriting headlines as headlines (i.e. not as prose), having taxi passengers say different things depending on their occupation, and so on. Takes me ages to re-insert my fixes into the game every time I update it...

**Comment by zonkmachine** [comment]
I'm happy to translate more strings.

**Comment by Keeper1st** [comment]
Here are my current language files. http://www.keeper1st.com/pioneer/data.zip Not all have had corrections or changes, but most. Some are minor punctuation like fixing improper comma usage. Some are actual spelling issues or cleaning up awkward prose (e.g. "No more space for more cargo"). Some are more important like not assuming the player is male. Some are rewrites (e.g. taxi passengers either for more variety in what they say or in making the way they talk reflect their profession). Headlines are rewritten as headlines and not prose (e.g. instead of "The gas algae harvest in the {system} system is above average" it becomes "Gas algae harvest in {system} above average"). One is a little nod to modern air traffic control: adding "Good day" to one of the sign-offs from a station as you leave their space.

**Pull request merged**